arches_references/datatypes/datatypes.py

```diff
@@ -119,19 +157,48 @@ def transform_exception(e):
         }
 
     def transform_value_for_tile(self, value, **kwargs):
-        list_id = kwargs.get("controlledList")
-        value = self.serialize(value)
-        if (
-            isinstance(value, list)
-            and isinstance(value[0], dict)
-            and "value" in value[0]
-        ):
-            value = value[0]["value"]
-        if isinstance(value, str):
-            found_item = self.lookup_listitem_from_label(value, list_id)
+        if value is None:
+            return None
+        if not isinstance(value, list):
+            value = [value]
+
+        # Pre-process to discard display values generated by to_representation().
+        pre_processed_values = []
+        for single_value in value:
+            if isinstance(single_value, dict) and (
+                list_item_id := single_value.get("list_item_id")
+            ):
+                pre_processed_values.append(list_item_id)
+            else:
+                pre_processed_values.append(single_value)
```

Comment:
This probably belongs in the serialization layer instead of here, but I'm just getting a first iteration out to unblock folks.